### PR TITLE
Add Beta label to Flink product.

### PR DESCRIPTION
**Pull request by anelook**
Added label to 
 - Flink card information on the landing page
 - Flink main menu item
 - corrected styles for the menu, so that visually they look on the same line

### CODE REVIEW

File: docs/products/flink/index.rst [hunk 1]
```diff
@@ -1,5 +1,5 @@
-Aiven for Apache Flink
-=======================
+Flink :badge:`beta,cls=badge-secondary text-black badge-pill`
```
Comment (lornajane) [inline docs/products/flink/index.rst:1]: Would it still fit if it was "Apache Flink" ?

Comment (ftisiot) [reply docs/products/flink/index.rst:1]: Yep, seen that we used Apache Kafka, we should do the same for Flink

**Comment by lornajane** [comment]
Can we take another look at this one? There's a LOT of commits and some feedback waiting @anelook

**Comment by anelook** [comment]
Updated the name, it fits into the space. However, it is not very consistent with other services, where we have short titles.

**Review by lornajane — APPROVED** [review]
Looks good, and I don't think it's too long

**Pull request merged**